### ISSUE TITLE
Compliance/Compliance Obligation Grid - internal user can view all compliance obligations for industry

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-compliance#28]
#### Description:

As an internal user, I can see the compliance obligation grid for each compliance obligation.

- [Design in Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?page-id=14307%3A1663&node-id=21296-1060&node-type=frame&viewport=1759%2C-4811%2C0.18&t=CZ8iNtq9kg6AOlwP-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=21296%3A1271&show-proto-sidebar=1)

#### Acceptance Criteria:

- [ ] Given I clicked on "Compliance Administration" on the [dashboard](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=21296-1271&t=IvFnxqUXsk08lg93-4), then I am brought to the [Compliance Summaries grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=21296-1060&t=IvFnxqUXsk08lg93-4)
- [ ] The grid shows the compliance year, operator name, operation name, obligation ID, excess emissions, outstanding balance, compliance status, and a link to view details.

#### Development Checklist:

- [ ] In Action cells, implement logic for internal users. If a decision has been made, display "View Details" instead of "Review Issuance of Credits".
- [ ] In Action cells, implement logic for external users. Once the request has been submitted—regardless of whether it has been approved or not—display "View Details" instead of "Request Issuance of Credits".
- [ ] Update the list of allowed cas roles to include cas_admin and cas_view_only
- [ ] Add the "Compliance Administration" header to the compliance obligation grid
- [ ] Add the navigation tabs section with the following tabs: "Compliance Summaries", "Imposed Penalties", "Invoices"
- [ ] Update the default value in the ComplianceEarnedCredit model for the issuance_status field from CREDITS_NOT_ISSUED to ISSUANCE_REQUESTED.
- [ ] Update tests

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by #45 
- Blocking

**Comment by nanyangpro** [comment]
Link to design is updated in the ticket description

**Comment by vesselak** [comment]
@nataliepiguliak Is this work merging soon? I see that both Sepehr and Shon have reviewed the PR

**Comment by nataliepiguliak** [comment]
@vesselak yes, will merge shortly

**Comment by fviduya** [comment]
I have tested this in the DEV environment, [here is the test document for reference](https://buttoninc-my.sharepoint.com/:w:/g/personal/francesca_button_is/ERCCN28DMKFLrL5C_VlawW8BGRA9KAHSrCgnkPZ532pq2Q?e=WxEDlF).

I can verify that:
- [x] Given I clicked on "Compliance Administration" on the [dashboard](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=21296-1271&t=IvFnxqUXsk08lg93-4), then I am brought to the [Compliance Summaries grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=21296-1060&t=IvFnxqUXsk08lg93-4)
- [x] The grid shows the compliance year, operator name, operation name, obligation ID, excess emissions, outstanding balance, compliance status, and a link to view details. - **Noting that Penalty Status is also displayed in the grid, hi @vesselak is that okay for internal users to see as well? It's not in the AC but it's in DEV and in the wireframe as well**
- [x] The action name for a report with earned credits is `Review Credits Issuance Request` similar to the name in the wireframe
- [x] The action name for a report with excess emission and no excess/no credit is `View Details` similar to the name in the wireframe

Moving to Pending PO for review of test. Thank you!

**Comment by vesselak** [comment]
Reviewed testing documentation and accepting ticket with the format deficiencies for some of the numbers
